main/src/ui/conversation_selector/conversation_selector_row.vala

```diff
@@ -130,14 +130,34 @@ public class ConversationSelectorRow : ListBoxRow {
                     MessageItem message_item = last_content_item as MessageItem;
                     Message last_message = message_item.message;
 
-                    if (conversation.type_ == Conversation.Type.GROUPCHAT) {
-                        nick_label.label = Util.get_participant_display_name(stream_interactor, conversation, last_message.from, true) + ": ";
+                    string body = last_message.body;
+                    bool me_command = body.has_prefix("/me");
```

Comment:
The correct prefix according to [XEP-0245](https://xmpp.org/extensions/xep-0245.html) is `/me ` (with a space).